backend/dal/Migrations/v0.2.13.3/Up/PostUp/01-History.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE PIMS_LEASE_HIST ALTER COLUMN LEASE_PAY_RVBL_TYPE_CODE NVARCHAR(20) NULL
```

Comment:
There is no down for this since this is essentially a bug in the original schema. The _HIST tables should have the same null constraints as the underlying table.